GPRegistryPolicyParser.psm1

```diff
@@ -303,57 +260,56 @@ Reads registry policies from a list of entries.
 Reads registry policies from a list of entries and returns an array of GPRegistryPolicies.
 
 .PARAMETER Division
-Specifies the division from which the registry entries will be read.
+Specifies the division from which the registry entries will be read: HKLM for Local Machine
+and HKCU for Current User.
 
 .EXAMPLE
-C:\PS> Read-RegistryPolicies -Division "LocalMachine"
+C:\PS> Read-RegistryPolicies -Division "HKLM"
 
 .EXAMPLE
-C:\PS> Read-RegistryPolicies -Division "LocalMachine" -Entries @('Software\Policies\Microsoft\Windows', 'Software\Policies\Microsoft\WindowsFirewall')
+C:\PS> Read-RegistryPolicies -Division "HKLM" -Entries @('Software\Policies\Microsoft\Windows', 'Software\Policies\Microsoft\WindowsFirewall')
 #>
 Function Read-RegistryPolicies
 {
     [OutputType([Array])]
     param (
 
-        [ValidateSet("LocalMachine", "CurrentUser", "Users")]
+        [ValidateSet("HKLM", "HKCU")]
```

Comment:
All changes in function signatures could break GPRegistryPolicy module.

---
In reply to: [121241843](https://github.com/PowerShell/GPRegistryPolicyParser/pull/1#discussion_r121241843) [](ancestors = 121241843)